app/src/app/[locale]/health/page.tsx

```diff
@@ -1,3 +1,10 @@
 export default function Page() {
-  return <>healthy</>;
+  return (
+    <>
+      <head>
+        <title>Health Check</title>
+      </head>
+      <div>healthy</div>
+    </>
+  );
```

Comment:
A little bit of a separate issue but this fixes an actual a11y issue

Reply:
We have a localization library in this repo - is there a reason not to use it here?

Reply:
Sure! Just added it . 

I assume because we're not really refining tickets here we're not splitting them up as we might on our project - first inclination is to actually break a task like that out.

Reply:
Do you mean to say that you think including content strings using the localization library should be a separate task ticketed separately?

Reply:
I think so because it's getting a bit off track of the intent of this PR